src/Footer/Footer.tsx

```diff
@@ -10,7 +10,7 @@ export default function Footer({ columns, copyrightText }) {
   }
 
   return (
-    <FooterContainer>
+    <FooterContainer data-component-name="Footer/Footer">
```

Comment:
what is the purpose of double Footer/Footer?
```suggestion
    <FooterContainer data-component-name="Footer">
```

Reply:
This `data-component-name` attribute is added to to make better discoverability of components when user will want to override them with `eject-script`. 
So when users will call `yarn eject-component ...` they will pass `Footer/Footer` as an argument instead of just `Footer` which is a sort of namespace for footer components.